manifests/init.pp

```diff
@@ -27,15 +27,29 @@
 #   The user to use when performing an aptly command
 #   Default: 'root'
 #
+# [*aptly_repos*]
+#   Hash of aptly repos which is passed to aptly::repo
+#   Default: {}
+#
+# [*aptly_mirrors*]
+#   Hash of aptly mirrors which is passed to aptly::mirror
+#   Default: {}
+#
 class aptly (
   $package_ensure = present,
   $config = {},
   $repo = true,
   $key_server = undef,
   $user = 'root',
+  ### START Hiera Lookups ###
```

Comment:
I'm not sure these comments are strictly necessary because any of the parameters can be provided by hiera lookups.